### PR TITLE
Add a direct access to tables, for stronger typing.

### PR DESCRIPTION
**Pull request by zozoens31**
This change is [Reviewable](https://reviewable.io/reviews/josephbestjames/airtable.py/33)

### CODE REVIEW

File: airtable/airtable.py [hunk 1]
```diff
@@ -159,3 +159,34 @@ def delete(self, table_name, record_id):
...
+                  returned by the API using an OrderedDict, but you can switch
+                  to a simple dict if you prefer.
+        """
+        self.table_name = table_name
+        if isinstance(base_id, Airtable):
+            self._client = base_id
+            return
+        self._client = Airtable(base_id, *args, **kwargs)
+
+    def __getattr__(self, name):
```
Comment (pcorpet) [inline airtable/airtable.py:186]: Please add some tests.

File: airtable/airtable.py [hunk 2]
```diff
@@ -159,3 +159,34 @@ def delete(self, table_name, record_id):
         if check_string(table_name) and check_string(record_id):
             url = posixpath.join(table_name, record_id)
             return self.__request('DELETE', url)
+
+    def table(self, table_name):
+        return AirtableTable(self, table_name)
+
+
+class AirtableTable(object):
```
Comment (pcorpet) [inline airtable/airtable.py:167]: Please rename this simply `Table`

**Review by pcorpet — CHANGES_REQUESTED** [review]
Thanks for the suggestion, I do like this new API.

**Review by zozoens31 — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/josephbestjames/airtable.py/33#-:-MRTqvj5C3qzzPoNKaUf:bnfp4nl)* status: 0 of 5 files reviewed, all discussions resolved

**Review by pcorpet — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/josephbestjames/airtable.py/33#-:-MRTwICB10hHkkxb3dj5:bnfp4nl)* status: 0 of 5 files reviewed, all discussions resolved

**Review by pcorpet — APPROVED** [review]
> Reviewed 5 of 5 files at r2.
> *[Reviewable](https://reviewable.io/reviews/josephbestjames/airtable.py/33)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @zozoens31)

**Pull request merged**